src/containers/Tenant/Diagnostics/Consumers/columns/columns.tsx

```diff
@@ -23,7 +29,28 @@ export const columns: Column<IPreparedConsumerData>[] = [
         name: CONSUMERS_COLUMNS_IDS.CONSUMER,
         header: CONSUMERS_COLUMNS_TITILES[CONSUMERS_COLUMNS_IDS.CONSUMER],
         align: DataTable.LEFT,
-        render: ({row}) => row.name || '-',
+        render: ({row}) => {
+            if (!row.name) {
+                return '–';
+            }
+
+            const queryParams = qs.parse(location.search, {
+                ignoreQueryPrefix: true,
+            });
+
+            return (
+                <Link
+                    className="link"
```

Comment:
what is this classname?
uikit classname is `yc-link`
`InternalLink` uses it

Reply:
It is the classname defined in mixins

Reply:
Changed to `InternalLink`

Reply:
i don't think it is used anywhere directly, save the `@extend` statements

my issues with this:
- classname is determined without bem-cn-lite, which is used everywhere else (the `'error'` classname has the same issue btw)
- styles for this classname do not depend on uikit link styles, and can diverge

basically, my suggestion is to use `InternalLink`
it encapsulates uikit link styles combined with react-router-dom link behavior

Reply:
that took me too long to phrase